doc/refman/RefMan-pre.tex

```diff
@@ -1335,6 +1335,157 @@ \section*{Credits: version 8.7}
+system.
+
+The OPAM repository for {\Coq} packages has been maintained by Guillaume
+Melquiond, Matthieu Sozeau, Enrico Tassi with contributions from many
+users. A list of packages is available at \url{https://coq.inria.fr/opam/www/}.
+
+TODO Outdated?
+Packaging tools and software development kits were prepared by Michael
+Soegtrop with the help of Maxime Dénès and Enrico Tassi for Windows, and
+Maxime Dénès for MacOS X. Packages are regularly built on the
+Travis and GitLab continuous integration server.
+
+The 40 contributors for this version are Yves Bertot, Joachim
+Breitner, Tej Chajed, Arthur Charguéraud, Jacques-Pascal Deplaix, Maxime
+Dénès, Jim Fehrle, Yannick Forster, Gaëtan Gilbert, Jason Gross, Samuel
+Gruetter, Thomas Hebb, Hugo Herbelin, Jasper Hugunin, Emilio Jesus
+Gallego Arias, Ralf Jung, Johannes Kloos, Matej Košík, Robbert Krebbers,
+Tony Beta Lambda, Vincent Laporte, Pierre Letouzey, Farzon Lotfi,
+Cyprien Mangin, Guillaume Melquiond, Raphaël Monat, Carl Patenaude
+Poulin, Pierre-Marie Pédrot, Matthew Ryan, Matt Quinn, Sigurd Schneider,
+Bernhard Schommer, Matthieu Sozeau, Arnaud Spiwack, Paul Steckler,
+Enrico Tassi, Anton Trunov, Martin Vassor, Vadim Zaliva and Théo
+Zimmermann.
+
+The development process was coordinated by Matthieu Sozeau. Maxime Dénès
+was in charge of the release process. The maintainer of this release
+is ???.
+
+Many power users helped to improve the design of the new features via
+the bug tracker, the pull request system, the {\Coq} development mailing
+list or the coq-club mailing list. Special thanks to the users who
+contributed patches and intensive brain-storming and code reviews,
+starting with Jason Gross, Ralf Jung, Robbert Krebbers, Amin Timany,
+..???.  It would however be impossible to mention exhaustively the names
+of everybody who to some extent influenced the development.
+
+Version 8.8 is the third release of {\Coq} developed on a time-based
+development cycle. Its development spanned 6 months from the release of
+{\Coq} 8.7 and was based on a public road-map. It attracted many
+external contributions. Code reviews and continuous integration testing
+were systematically used before integration of new features, with an
+important focus given to compatibility and performance issues.
+
+Coq Enhancement Proposals (CEPs for short) and open pull-requests
+discussions were used to discuss publicly the new features. The
+developer's Coq Working Groups, which are systematically streamed on the
+Coq YouTube channel, are moving towards a faster frequency
+and experimenting videoconferencing technology, reflecting the
+increasingly multi-site nature of Coq development.
+
+The {\Coq} consortium, an organization directed towards users and
+supporters of the system, is now running and employs Maxime Dénès.
+The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
+
+The developers of {\Coq} are also authors of plugins that are highly
```

Comment:
Do we want this section in the first place? It is not about Coq

Reply:
Which one, the Consortium or the related tools?

Reply:
The plugins part is off topic in my opinion for two reasons:
- not about Coq (we are releasing Coq alone, with no plugins attached)
- the selection is arbitrary, there are plugins not part of the list being developed (and libraries) and making Coq great that are not there

The consortium part: I don't have an opinion

Reply:
I tend to agree that the plugins developed by the same developers are a bit off-topic.

Reply:
Alright, I just kept the Coq consortium paragraph.